app/assets/javascripts/modules/form-name.js

```diff
@@ -28,7 +28,7 @@ window.moj.Modules.FormName = {
     var self = this;
 
     if (self.$identifier.val().length > 0 && self.$unknown.is(':checked')) {
-      self.$unknown.attr('checked', false);
```

Comment:
No, the problem was that when the box was unchecked programattically the label remained highlighted. 

This should have been in a separate PR though – it's nothing to do with line lengths... my bad.